Build fails: undefined reference to `OpenAL::OpenAL()'

It was broken by 28c2298.
@sudden6 

Building using qtox.pro on FreeBSD.

```
linking qtox
audio.o: In function `Audio::getInstance()':
src/audio/audio.cpp:(.text+0x3e): undefined reference to `OpenAL::OpenAL()'
src/audio/audio.cpp:(.text+0x45): undefined reference to `OpenAL::~OpenAL()'
```

--- a/qtox.pro
+++ b/qtox.pro
@@ -424,6 +424,7 @@ RESOURCES += res.qrc \
 
 HEADERS  += \
     src/audio/audio.h \
+    src/audio/backend/openal.h \
     src/chatlog/chatline.h \
     src/chatlog/chatlinecontent.h \
     src/chatlog/chatlinecontentproxy.h \
@@ -545,6 +546,7 @@ HEADERS  += \
 
 SOURCES += \
     src/audio/audio.cpp \
+    src/audio/backend/openal.cpp \
     src/chatlog/chatline.cpp \
     src/chatlog/chatlinecontent.cpp \
     src/chatlog/chatlinecontentproxy.cpp \

